src/plugins/overlay/reports/a11y-2.json

```diff
@@ -0,0 +1,764 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2022/acr-new-wcag-sc.json-ld",
+
+    "@id": "https://wet-boew.github.io/wet-boew/demos/overlay/reports/a11y-2.json",
+    "id": "2",
+    "@type": "earl:Assertion",
+    "earl:test": ["WCAG20", "WCAG21", "WCAG22"],
+
+    "subjectUrl": "https://wet-boew.github.io/wet-boew/demos/overlay/overlay-en.html",
+
+    "assertedBy": {
+        "name": "Service Canada - Principal Publisher",
+        "@id": "https://github.com/ServiceCanada",
+        "@type": ["earl:Assertor", "foaf:Organization"],
+        "producedBy": {
+            "name": "Hani Ahmed (Github: @heansih)",
+            "@id": "https://github.com/heansih",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+    "date": "2023-01-06",
+
+    "assessment": "Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2",
+    "notes": "",
+    "score": "na",
+
+    "results": [{
+            "label": "SC 1.1.1 Non-text content (Level A)",
+            "earl:test": "WCAG22:non-text-content",
+            "outcome": "Fail",
+            "earl:outcome": "earl:fail",
+            "notes": "Title attribute, text within <a> element. Role attribue is missing within <a> element.",
+            "techniques": [
+                "WCAG21Tech:HTML/H91",
+                "WCAG21Tech:general/G94",
+                "WCAG21Tech:general/G95",
+                "WCAG21Tech:general/G82",
+                "WCAG21Tech:failures/F3",
+                "WCAG21Tech:failures/F13",
+                "WCAG21Tech:failures/F20"
```

Comment:
Note: We have recently changed/clarified the guidance regarding the use of the "techniques" property

Please only list the technique that we observed it was fully applied and the techniques being discussed in the notes, like applicable failure and sufficient technique that should be applied.